### DISCUSSION TITLE
Edit interaction response with embeds

### DISCUSSION THREAD

**Discussion by etr-dev** [kiki7000/discord.py-components#98] [Q&A]
How do I edit a response to an interaction by sending embeds? Currently I can edit it with string content but not embedded messages.

I need to be able to respond on an interaction like "fetching data..." then once the data is fetched edit the fetching data message to display the data in an embed.

**Comment by Damego** [comment]
```python
embed = Embed(...)
<Interaction>.edit_origin(embed=embed)
```
